device/iothub_client_python/src/iothub_client_python.cpp

```diff
@@ -53,6 +53,7 @@
 #endif
 
 #define VERSION_STRING "1.4.5"
+#define PYTHON_PRODUCT_INFO "python/" VERSION_STRING
```

Comment:
>PYTHON_PRODUCT_INFO [](start = 8, length = 19)

Just in general - is there any desire for PY callers themselves to be able to set this value?  Same way C SDK lets you append stuff, should there be a python_object.set_option("Product_info") that appends 3rd party logic onto what you're building up here?

Reply:
i'm fine with this :) 

---
In reply to: [244873976](https://github.com/Azure/azure-iot-sdk-python/pull/249#discussion_r244873976) [](ancestors = 244873976)